init.php

```diff
@@ -178,12 +175,17 @@ public function api_version()
 
     private function isDomainEnabled($feedUri)
     {
-        $enabledDomains = $this->host->get($this, STORAGE_ENABLED_DOMAINS, array());
+        $enabledDomains = $this->host->get($this, "STORAGE_ENABLED_DOMAINS", array());
         $host = parse_url($feedUri, PHP_URL_HOST);
 
         if (strpos($host, "www.") === 0 && in_array(str_replace("www.", "", $host), $enabledDomains)) {
             return true;
         }
         return in_array($host, $enabledDomains);
     }
+
+    public function csrf_ignore($method) {
+        return true;
+    }
```

Comment:
It is safer to only ignore CSRF on specific requests. This function should only return true if `$method` is `proxy`. All other method should check CSRF tokens normally.